src/framework/preferences/js/PrefsEditor.js

```diff
@@ -177,22 +154,27 @@ var fluid_2_0 = fluid_2_0 || {};
         }
     });
 
-    fluid.prefs.resourceLoader.resolveTemplates = function (that) {
-        var mapped = fluid.transform(that.options.templates, that.transformURL);
+    fluid.prefs.resourceLoader.resolveResources = function (that) {
+        var mapped = fluid.transform(that.options.resources, that.transformURL);
 
         return fluid.transform(mapped, function (url) {
             return {url: url, forceCache: true};
         });
     };
 
-    fluid.prefs.resourceLoader.loadTemplates = function (that, resources) {
-        delete resources.expander;   // A work-around for FLUID-5117
+    fluid.prefs.resourceLoader.loadResources = function (that, resources, transformFn) {
         fluid.fetchResources(resources, function () {
-            that.resources = resources;
+            that.resources = transformFn(resources);
             that.events.onResourcesLoaded.fire(resources);
         });
     };
 
+    fluid.prefs.resourceLoader.parseMessages = function (resources) {
```

Comment:
More efficient to have this done by the jQuery ajax machinery - can we arrange to have this done by a suitable "dataType" option passed through as we do the fetch? Then we can lose the whole "transformResources" workflow